front/src/modules/ui/table/editable-cell/types/EditableChip.tsx

```diff
@@ -67,6 +67,7 @@ export function EditableCellChip({
         <StyledInplaceInput
           placeholder={placeholder || ''}
           autoFocus
+          autoComplete="off"
```

Comment:
Thank you!
you would also need to make this modification in other types of @/ui/table/editable-cell and @/ui/editable-fields